tools/scripts/used-headers.txt

```diff
@@ -39,9 +39,11 @@ cstdio
 cstdlib
 cstring
 ctime
+fstream
 initializer_list
 iomanip
 iostream
+istream
```

Comment:
Those are already indirectly used via `cpptoml`.

Reply:
@elBoberido Jup, that's what I had in mind.